vslib/src/sai_vs_switch_BCM56850.cpp

```diff
@@ -1210,3 +1210,119 @@ sai_status_t refresh_read_only_BCM56850(
 
     return SAI_STATUS_NOT_IMPLEMENTED;
 }
+
+sai_status_t vs_create_port_BCM56850(
```

Comment:
please reuse logic that creates ports in init function, extract that code to separate function and reuse